_config.yml

```diff
@@ -0,0 +1,15 @@
+# Setup
+title:            CoMPAS Architecture
+tagline:          'Architecture of the CoMPAS project'
+
+paginate:       5
+
+# Custom vars
+version:          0.0.1
+
+github:
+  repo:           https://github.com/com-pas/compas-architecture
+
+github_username:  Flurb
```

Comment:
Is Flurb still relevant in the settings? This is not a personal page.

Reply:
I guess that the username would be: com-pas

Reply:
Correct, it should be changed.
It was purely meant for debugging.